src/WebJobs.Script/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -33,6 +33,16 @@ public static bool IsHttpInAndOutFunction(this FunctionMetadata metadata)
             return false;
         }
 
+        public static bool IsHttpInFunction(this FunctionMetadata metadata)
```

Comment:
i know you're following the other method naming -- but it seems more accurate to call this `IsHttpTriggerFunction`...